### DISCUSSION TITLE
Uploading Assets for MTO Orders

### DISCUSSION THREAD

**Discussion by bergstar** [etsy/open-api#1204] [Q&A]
When customers place an MTO (Made to Order) order, how can I upload an assets file and fulfill this order?

I've explored the Receipt/Receipts documentation but wasn't able to find anything related to file upload.

Should I upload it to the original MTO listing? If so, how do I track different orders?

**Comment by Gareth064** [comment]
If you mean you want to send the customer a digital file after the order has been placed, then this can only be done using the Etsy Message system, or by maybe email them if you have their email address.

The only way to provide as a digital download is if the finished file is somehow already on the listing before they purchase.

**Reply by bergstar** [reply]
Thank you so much for your answer. So in practical terms, if something like this should be automated it will be using Puppeteer?

**Comment by Gareth064** [comment]
I'm going to say yes based on the quick sentence I read about what Puppeteer was 🙂


Kind Regards

Gareth Doherty



It’s nice to be important,

But it’s more important to be nice.



________________________________
